franck-bereket.html

```diff
@@ -0,0 +1,11 @@
+<main id="main">
```

Comment:
I think it would be better if you use the DOCTYPE tag in the first line and put your content in the HTML tag so that anyone who wants to see how your project looks in a browser can do it locally.